src/main/java/web/presentation/ReservationController.java

```diff
@@ -0,0 +1,90 @@
+package web.presentation;
+
+import java.net.URI;
+import java.sql.Date;
+import java.sql.Time;
+import java.util.List;
+import javax.sql.DataSource;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import web.domain.Reservation;
+import web.domain.Theme;
+import web.dto.request.ReservationRequestDTO;
+import web.dto.response.ReservationResponseDTO;
+import web.exception.DuplicatedReservationException;
+import web.exception.NoSuchReservationException;
+
+@RestController
+@RequestMapping("/reservations")
+public class ReservationController {
+
+    private final JdbcTemplate jdbcTemplate;
+    private final SimpleJdbcInsert insertActor;
+    private final Theme defaultTheme = new Theme("워너고홈", "병맛 어드벤처 회사 코믹물", 29_000);
+
+    public ReservationController(final JdbcTemplate jdbcTemplate, final DataSource dataSource) {
+        this.jdbcTemplate = jdbcTemplate;
+        this.insertActor = new SimpleJdbcInsert(dataSource)
+                .withTableName("reservation")
+                .usingGeneratedKeyColumns("id");
+    }
+
+    @PostMapping("")
```

Comment:
쌍따옴표를 제거하면 어떻게 달라지나요?
```suggestion
    @PostMapping()
```

Reply:
`@PostMapping` 이렇게 처리하도록 하겠습니다~